### ISSUE TITLE
Error with constraint names on postgres

### ISSUE THREAD

**Issue by andrew-w-ross** [typeorm/typeorm#108]
On postgres there is a break on db syncing if the same column name is defined on multiple tables with a unique constraint. 

Take the following models as an example :
```typescript
@Table()
export class A{
    @PrimaryGeneratedColumn()
    id:number;

    @Column({unique : true, nullable:false})
    name:string;
} 

@Table()
export class B{
    @PrimaryGeneratedColumn()
    id:number;

    @Column({unique : true, nullable:false})
    name:string;
} 
```
The creation of table b will break as `uk_name` is already defined in table a. Perhaps add the table name into the constraint name in the form uk_[table]_[column]. It might be beneficial to add unique constraint names to the naming strategy interface.

@pleerock If you need help just choose a solution and I'll make a pr.

**Comment by pleerock** [comment]
Not sure how to make it in the naming strategy interface, but you can do it in the `PostgresQueryRunner` just replacing `uk_ + columnName`  to `uk- + tableName + _ columnName`. Feel free to create a PR ;)

**Comment by pleerock** [comment]
if you are not able to make a PR, please let me know, I'll fix it on my own.

**Comment by andrew-w-ross** [comment]
@pleerock  Hey if it's urgent I would suggest you do it but if it can wait a day I'd like to do a pr. The setup time for this project is quite a bit so it killed my monday night os dev time.

**Comment by pleerock** [comment]
btw today I made setup much much better. I have removed problematic packages from package.json at all (mariadb and oracle). All test configuration is now read from `ormconfig.json`, all you need to do is copy `ormconfig.json.dist` file into `ormconfig.json` and change with your own configuration there. Also I have added docker-compose file if you are using docker you can easily run using docker. Now it really should be a 5-minute setup.

**Comment by andrew-w-ross** [comment]
@pleerock Excellent, actually installing all those db's was taking forever. I thought I had to do it because node-gyp was always complaining about mariadb and oracle.

**Comment by pleerock** [comment]
fixed in #117
